demos/storybook/stories/drawer/with-basic-config.tsx

```diff
@@ -1,54 +1,57 @@
-import { Accessibility, Menu, NotificationsActive, Person, Today, Gavel, Settings } from '@material-ui/icons';
+import { Accessibility, Menu, NotificationsActive, Person, Today } from '@material-ui/icons';
 import { Drawer, DrawerBody, DrawerHeader, DrawerNavGroup, NavItem } from '@pxblue/react-components';
 import { boolean, text } from '@storybook/addon-knobs';
 import { StoryFnReactReturnType } from '@storybook/react/dist/client/preview/types';
 import React from 'react';
 import { WITH_MIN_PROPS_STORY_NAME } from '../../src/constants';
-import { DrawerStoryContext } from './util';
+import { useState } from '@storybook/addons';
+import { State, Store } from '@sambego/storybook-state';
+import { DrawerState } from './util';
 
-export const navGroupItems1: NavItem[] = [
-    {
-        title: 'Identity Management',
-        itemID: '1',
-        icon: <Person />,
-    },
-    {
-        title: 'Calendar',
-        itemID: '2',
-        icon: <Today />,
-    },
-    {
-        title: 'Accessibility',
-        itemID: '3',
-        icon: <Accessibility />,
-    },
-    {
-        title: 'Notifications',
-        itemID: '4',
-        icon: <NotificationsActive />,
-    },
-];
+const store: Store<DrawerState> = new Store<DrawerState>({
+    selected: '',
+});
```

Comment:
This doesn't actually do anything...you never read it and never change it (you're just using the local react state variable below).